operators/hack/kind/kind.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
Can we/should we move this to deployer? It seems to me that makefile would be also simplified if we did that.

Reply:
I would rather keep it in a shell for the moment and move it in an other PR ?

Reply:
Sounds good.